nncf/experimental/torch2/function_hook/graph_utils.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2024 Intel Corporation
```

Comment:
What do you think about creating nncf/experimental/torch2/function_hook/graph module?

Reply:
done